docs/xml-sitemaps.md

```diff
@@ -0,0 +1,71 @@
+# XML Sitemaps
+
+XML Sitemaps are a means of providing key content and additional meta data around the frequency of updates on your website to search engines.
+
+The functionality for this component is provided by the [MSM Sitemap](https://github.com/humanmade/msm-sitemap) plugin.
+
+The sitemaps follow the specification detailed on https://www.sitemaps.org/ and are supported by Google and all major search engines.
+
+The sitemaps are automatically listed in the site's [./robots-txt.md](robots.txt) file so you don't need to manually submit them to the [Google Search Console](https://search.google.com/search-console/) but you can resubmit them there and get diagnostic information there at any time.
+
+It is necessary to [verify the site with Google Search Console](https://support.google.com/webmasters/answer/9008080?hl=en) before you can access information about your site's search results performance. It is recommended to use the HTML file upload solution by comitting the file to your project's root directory.
```

Comment:
Could we add a platform config for this maybe?

Reply:
I guess the file is named specifically too -- it should still be possible we'd just have to generate the file at build time perhaps. But yeah, don't need to do it now, might just be worth tracking in an issue

Reply:
Yeah, I'll add an issue for that one too. A meta tag is also possible but it's better to rely on a stable URL that's still there even if PHP falls over for whatever reason.